Makefile

```diff
@@ -1,30 +1,29 @@
 ifeq ($(OS),Windows_NT)
   BIN_EXT := .exe
-  SCRIPT_EXT := .ps1
-  SUBSHELL := powershell -ExecutionPolicy Bypass -File
+  SCRIPT_EXT := .bat
+  PATHSEP := \\
+  SUBSHELL := cmd /C
 else
   BIN_EXT :=
   SCRIPT_EXT := .sh
+  PATHSEP := /
   SUBSHELL :=
 endif
+
 BIN := build-conan/build/Release/tflite_cpu$(BIN_EXT)
 
 .PHONY: tflite_cpu
 tflite_cpu: $(BIN)
 
-$(BIN): src/*
-	$(SUBSHELL) ./bin/build$(SCRIPT_EXT)
+$(BIN): conanfile.py src/*
+	$(SUBSHELL) bin$(PATHSEP)build$(SCRIPT_EXT)
 
 .PHONY: setup
 setup:
-	$(SUBSHELL) ./bin/setup$(SCRIPT_EXT)
+	$(SUBSHELL) bin$(PATHSEP)setup$(SCRIPT_EXT)
 
-module.tar.gz: tflite_cpu
-ifeq ($(OS),Windows_NT)
-	7z a -tgzip module.tar.gz $(BIN)
-else
-	tar -czvf module.tar.gz $(BIN)
-endif
+module.tar.gz: $(BIN) meta.json
+	$(SUBSHELL) bin$(PATHSEP)package$(SCRIPT_EXT) $(BIN) meta.json
```

Comment:
You might replace `$(BIN) meta.json` on this line with `$^`, which represents all files that this make target depends on (everything to the right of the `:` on the previous line). That way, if there are new dependencies in the future, you only need to add them once.

Reply:
I had that originally, for the reasons you suggest, but I was getting results that didn't make sense. I'll try it again and update it if works.

Reply:
Yes, working now. I suspect the issue I had earlier was with argument passing, not `$^`.